go.mod

```diff
@@ -314,7 +315,6 @@ require (
 	github.com/lufia/plan9stats v0.0.0-20211012122336-39d0f177ccd0 // indirect
 	github.com/magiconair/properties v1.8.5 // indirect
 	github.com/mailru/easyjson v0.7.6 // indirect
-	github.com/mattn/go-ieproxy v0.0.1 // indirect
```

Comment:
Oh, can this be removed from the go.sum then? And distdir files?

Reply:
explained above.